modules/dists/DimensionalDist2D.chpl

```diff
@@ -135,7 +135,7 @@ and distributes using block-cyclic distribution in the second dimension.
     for loc in MyLocales do on loc {
 
       // The ReplicatedDim specifier always accesses the local replicand.
-      // (This differs from how the ReplicatedDist distribution works.)
+      // (This differs from how the Replicated distribution works.)
```

Comment:
Actually, now I feel more sure.  I hadn't thought much about what this was saying, nor about the fact that we'd already changed what ReplicatedDist does prior to this PR.